melosys-eessi-app/src/test/java/no/nav/melosys/eessi/integration/pdl/PDLServiceTest.java

```diff
@@ -0,0 +1,103 @@
+package no.nav.melosys.eessi.integration.pdl;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.Set;
+
+import no.nav.melosys.eessi.integration.pdl.dto.*;
+import no.nav.melosys.eessi.models.person.PersonModell;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class PDLServiceTest {
+
+    @Mock
+    private PDLConsumer pdlConsumer;
+
+    private PDLService pdlService;
+
+    @BeforeEach
+    public void setup() {
+        pdlService = new PDLService(pdlConsumer);
+    }
+
+    @Test
+    void hentPerson_personMedFlereEndringerPåNavnFlereStatsborgerskap_nyesteEndringerForNavnAlleStatsborgerskapMappes() {
+        final String ident = "12345600000";
+
+        when(pdlConsumer.hentPerson(eq(ident))).thenReturn(lagPersonMedFlereEndringer());
```

Comment:
Ikke nødvendig å bruke `eq()` når det er alene eller med andre 'eq()'. "The resulting code is shorter and easier to read." (fra Sonar)